WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -29,9 +29,9 @@ private AddedWalletPageViewModel(IWalletSettingsModel walletSettings, WalletCrea
 
 	public WalletType WalletType { get; }
 
-	public string WalletName { get; }
+	public string? WalletName { get; }
```

Comment:
It cannot be null.

Reply:
I'm afraid that I can't do more than I've alread done. WalletCreationOptions works with nullable types and it's shared among many different scenarios. What do you think, @ichthus1604? IIRC you refactored this code.